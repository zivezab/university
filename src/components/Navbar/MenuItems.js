export const MenuItems = [
  {
    title: 'Home',
    url: '/',
    className: 'nav-links',
  },
  {
    title: 'Search',
    url: '/search',
    className: 'nav-links',
  },
  {
    title: 'Favorite',
    url: '/favorite',
    className: 'nav-links',
  },
  {
    title: 'Login / Register',
    url: '/login',
    className: 'nav-links',
  },
];
