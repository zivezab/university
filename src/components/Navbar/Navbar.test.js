import {shallow} from 'enzyme';
import toJSON from 'enzyme-to-json';
import React from 'react';

import Navbar from './Navbar';

test('matches snapshot', () => {
  const wrapper = shallow(<Navbar />);

  expect(toJSON(wrapper)).toMatchSnapshot();
});
