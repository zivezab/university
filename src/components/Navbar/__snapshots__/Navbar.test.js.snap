// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<nav
  className="NavbarItems"
>
  <h1
    className="navbar-logo"
  />
  <div
    className="menu-icon"
    onClick={[Function]}
  >
    <i
      className="fas fa-bars"
    />
  </div>
  <ul
    className="nav-menu"
  >
    <li
      key="0"
    >
      <Link
        className="nav-links"
        to="/"
      >
        Home
      </Link>
    </li>
    <li
      key="1"
    >
      <Link
        className="nav-links"
        to="/search"
      >
        Search
      </Link>
    </li>
    <li
      key="2"
    >
      <Link
        className="nav-links"
        to="/favorite"
      >
        Favorite
      </Link>
    </li>
    <li
      key="3"
    >
      <Link
        className="nav-links"
        to="/login"
      >
        Login / Register
      </Link>
    </li>
  </ul>
</nav>
`;
